<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Day3Window</class>
 <widget class="QDialog" name="Day3Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="day3_title_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Day 3: Filterable Random Recipe Generator!</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="mealtype_label">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Select Meal Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mealtype_box">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Breakfast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lunch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dinner</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dessert</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="cuisinetype_label">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Select Cuisine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cuisinetype_box">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>French</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Japanese</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thai</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Indian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mexican</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Canadian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Italian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chinese</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="generaterecipe_button">
         <property name="text">
          <string>Generate Recipe Link:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextEdit" name="recipe_textedit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
